### PR TITLE
Wrap EmptyStateCardView and EmptyStateScreenView inside AndroidView Composables

### PR DESCRIPTION
**Pull request by yamal-alm**
### :goal_net: What's the goal?
Wrap `EmptyStateCardView` and `EmptyStateScreenView` inside `AndroidView` Composables

### :construction: How do we do it?
- Wrap and expose each attribute of `EmptyStateCardView` and `EmptyStateScreenView` into two news `AndroidView` Composables
- One Catalog page per each component just like before

This composables should be migrated at some point into fully compose components

### ☑️ Checks
- [x] I updated the documentation (readmes, wikis, etc). If no need to update documentation explain why.
- [x] Tested with dark mode.
- [x] Tested with API 21.

### :test_tube: How can I test this?
- [x] [Download link](https://install.appcenter.ms/orgs/tuenti-organization/apps/mistica-catalog/distribution_groups/public)
- [x] Reviewed by Mistica design team
- [x] 🖼️ Screenshots/Videos

https://user-images.githubusercontent.com/47142570/153871637-a68ff0c6-9a04-4411-878d-f9bd5d4e5bf5.mov

### CODE REVIEW

File: catalog-compose/src/main/java/com/telefonica/mistica/compose/catalog/ui/common/DropDown.kt [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.alpha
+import androidx.compose.ui.unit.dp
+
+@Composable
+inline fun <reified T : Enum<T>> DropDown(
```
Comment (yamal-alm) [inline catalog-compose/src/main/java/com/telefonica/mistica/compose/catalog/ui/common/DropDown.kt:29]: Util class for a DropDown available inside catalog-compose package

File: catalog-compose/src/main/java/com/telefonica/mistica/compose/catalog/ui/common/OnBackPressedHandler.kt [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+import androidx.activity.compose.LocalOnBackPressedDispatcherOwner
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.DisposableEffect
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.rememberUpdatedState
+import androidx.compose.ui.platform.LocalLifecycleOwner
+
+@Composable
+fun OnBackPressedHandler(
```
Comment (yamal-alm) [inline catalog-compose/src/main/java/com/telefonica/mistica/compose/catalog/ui/common/OnBackPressedHandler.kt:13]: another util class inside compose-catalog for onBackPressed handling

**Review by AnaMontes11 — APPROVED** [review]
Well done!

**Review by gmerinojimenez — APPROVED** [review]
Nice job!!

**Pull request merged**